ExpressAPI/db.js

```diff
@@ -0,0 +1,38 @@
+const mysql = require('mysql');
+
+const db = mysql.createConnection({
+    host: process.env.DB_HOST,
+    user: process.env.DB_USER,
+    password: process.env.DB_PASS,
+    database: process.env.DB_NAME
+});
+
+db.connect(function(err){
+    if (err) throw err;
+    console.log("Connected to mySQL")
```

Comment:
you should have some logging as well. sent a link to log4j have a look at that. this could be a task apart of another ticket though